com.vrcfury.vrcfury/Editor/VF/Feature/DefaultAdditiveLayerFixBuilder.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using VF.Builder.Exceptions;
+using VF.Feature.Base;
+using VRC.SDK3.Avatars.Components;
+
+namespace VF.Feature {
+    public class DefaultAdditiveLayerFixBuilder : FeatureBuilder {
+        [FeatureBuilderAction(FeatureOrder.RemoveDefaultedAdditiveLayer)]
+        public void Apply() {
+            var descriptor = avatarObject.GetComponent<VRCAvatarDescriptor>();
+
+            for (int i=0; i<descriptor.baseAnimationLayers.Length; i++) {
```

Comment:
If you iterate over VRCAvatarUtils.GetAllControllers().Where(c => c.type == Additive && c.isDefault), you can call c.set(null).

Reply:
Looks like I woke up a bit late to alter it haha. I wasn't aware of this utility function.